README.md

```diff
@@ -118,6 +118,8 @@ System properties are preferred over environment variables. The following system
 | `kubernetes.keystore.passphrase` / `KUBERNETES_KEYSTORE_PASSPHRASE`                                             |                                                                                                                                          |                                                       |
 | `kubernetes.backwardsCompatibilityInterceptor.disable` / `KUBERNETES_BACKWARDSCOMPATIBILITYINTERCEPTOR_DISABLE` | Disable the `BackwardsCompatibilityInterceptor`                                                                                          | `true`                                                |
 | `no.proxy` / `NO_PROXY`                                                                                         | comma-separated list of domain extensions [proxy](http://www.gnu.org/software/wget/manual/html_node/Proxies.html) should not be used for |                                                       |
+| `http.proxy` / `HTTP_PROXY`                                                                                     | URL to the [proxy](http://www.gnu.org/software/wget/manual/html_node/Proxies.html) for HTTP requests                                     |                                                       |
+| `https.proxy` / `HTTPS_PROXY`                                                                                   | URL to the [proxy](http://www.gnu.org/software/wget/manual/html_node/Proxies.html) for HTTPS requests                                    |                                                       |
```

Comment:
Should at least include a link to the FAQ entry.
The behavior of these ENV variables, system properties is not the same as the behavior of the rest of entries in the list. This should be highlighted maybe with a `[(*special order of precedence)](#link-to-faq)` or something like that.